src/components/Garden/Feed/Icons.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
```

Comment:
why are we doing this instead of using the 1hive ui component?

Reply:
Hey @rperez89, since the 1hive-ui types has some problems, migrating some files to typescript is impossible without adding some redundant and unnecessary props to some of the components ( in this case the majority of the icons ). When the PR from Felipe that make this icons optional, this file will be removed. I will add a TODO in this file.

Reply:
I think that Fix make all options its merged some time ago, just need bump the 1hive-ui version.

More details here - https://github.com/1Hive/1hive-ui/pull/28

Version 1.0.17, you can check if that works to you @tmoutinho ?

Reply:
Yes I will check it.

Reply:
Upgraded 1hive-ui to v1.0.17, and removed the Icons.js folder. All good.

Reply:
The file was removed and the 1hive-ui version was upgraded. With the help of @kamikazebr